features/step_definitions/note_steps.js

```diff
@@ -89,7 +89,7 @@ Then(
     'note {string} has the image URL {string}',
     function (noteName, expectedImageUrl) {
         const object = this.objects[noteName];
-        assert.equal(object.attachment.url, expectedImageUrl);
-        assert.equal(object.attachment.type, 'Image');
+        assert.equal(object.attachment[0].url, expectedImageUrl);
```

Comment:
Maybe first assert `object.attachment` exists, and has 1 entry before accessing directly? I know we get the same result, but the error message will be a little less cryptic

Reply:
Added assert on `object.attachment`. https://github.com/TryGhost/ActivityPub/pull/646/commits/82ee3c72e4651249a908eed0e74744edf437607a